js/package.json

```diff
@@ -3,17 +3,16 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
-    "flarum-webpack-config": "0.1.0-beta.16.2",
+    "flarum-webpack-config": "^1.0.0",
     "webpack": "^4.44.2",
-    "webpack-cli": "^3.3.12"
+    "webpack-cli": "^4.8.0"
   },
   "scripts": {
     "dev": "webpack --mode development --watch",
     "build": "webpack --mode production",
     "lint": "prettier --single-quote --trailing-comma none --print-width 120 --tab-width 4 --write src"
   },
   "devDependencies": {
-    "flarum": "0.1.0-beta.16",
```

Comment:
Why is this dependency removed?

Reply:
No longer required :)

Reply:
I believe it provided intellisense for Flarum, why is it not required anymore?